Crud.php

```diff
@@ -0,0 +1,271 @@
+<?php
+    use Doctrine\ORM\Tools\Pagination\Paginator;
+    require_once('src/Number.php');
+
+    /**
+     * Class to handle main actions(CRUD) of web app
+     */
+    class Crud {
+        //Object for manage doctrine entities
+        private $entityManager;
+
+        const PER_PAGE = 5; // Const for paginations(items per page)
+        const ENTITY_NAME = "Number"; // NAme of entity this controller handle
+        const ENTITY_PATH = __DIR__."/src/".self::ENTITY_NAME; // PAth to entity file
+        const ENTITY_ALIAS = "src\Number"; // Alias for entity
+
+        /**
+         * Method for creating of class
+         * 
+         * Takes entyty manager instance to work with doctrine
+         * entity manager instance is generated inside bootstrap file
+         * 
+         * @param entityManager
+         */
+        public function __construct($entityManager) {
+            $this->entityManager = $entityManager;
+        }
+
+        /**
+         * Adds new number
+         * 
+         * @param string slug
+         * @param string title
+         * @param int number
+         * @param string text
+         * @param string transcription
+         * 
+         * @return bool
+         */
+        public function createNumber(string $slug, string $title, int $number, string $text, string $transcription):bool {
+            $numberEntity = new Number();
+            $numberEntity->setSlug($slug);
+            $numberEntity->setTitle($title);
+            $numberEntity->setNumber($number);
+            $numberEntity->setText($text);
+            $numberEntity->setTranscription($transcription);
+            $numberEntity->setDate(new DateTime());
+            $this->entityManager->persist($numberEntity);
+            $this->entityManager->flush();
+
+            return true;
```

Comment:
what is the purpose of return true;?

Reply:
I was thinking that it would be better if we have returning data from function.